### PR TITLE
Improve accessibility for welcome/home page and dashboard

### LINKED ISSUE #3148: Comply with WCAG A/AA accessibility levels: welcome page
See https://docs.google.com/document/d/1M2tzjyHLzE8oRXLS3KAJ2n_GO9l-0QKq/edit?rtpof=true

### LINKED ISSUE #3150: Comply with WCAG A/AA accessibility levels: dashboard
See https://docs.google.com/document/d/1M2tzjyHLzE8oRXLS3KAJ2n_GO9l-0QKq/edit?rtpof=true

### PR DESCRIPTION
**Pull request by mjgiarlo**
# Why was this change made? 🤔

Fixes #3148
Fixes #3150

* Remove WCAG A/AA and aria violations from welcome page
* Remove WCAG A/AA and aria violations from home page
* Address accessibility errors in welcome & home pages identified by axe and wave plugins
* Eliminate accessibility violations on dashboard page

# How was this change tested? 🤨

- [x] CI
- [x] QA/stage

# Does your change introduce accessibility violations? 🩺

No.

### CODE REVIEW

File: app/components/citation_modal_component.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable modal-lg">
     <div class="modal-content">
       <div class="modal-header">
-        <h5 class="modal-title" id="citationModalHeader"></h5>
+        <h5 class="modal-title" id="citationModalHeader">Copy citation</h5>
```
Comment (mjgiarlo) [inline app/components/citation_modal_component.html.erb:5]: Empty headings bad.

File: app/components/dashboard/collection_component.html.erb [hunk 2]
```diff
@@ -23,8 +23,14 @@
...
     <tbody>
-      <% visible_deposits.take(MAX_DEPOSITS_TO_SHOW).each do |work| %>
-        <%= render "dashboards/collection_summary_row", work: work %>
+      <% if visible_deposits.any? %>
+        <% visible_deposits.take(MAX_DEPOSITS_TO_SHOW).each do |work| %>
+          <%= render "dashboards/collection_summary_row", work: work %>
+        <% end %>
+      <% else %>
+        <tr>
+          <td colspan="7" class="text-center">No visible deposits</td>
```
Comment (mjgiarlo) [inline app/components/dashboard/collection_component.html.erb:32]: Empty table bodies bad.

File: app/components/dashboard/in_progress_row_component.html.erb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <tr>
   <td>
     <%= work_link %>
-    <%= helpers.turbo_frame_tag dom_id(work, :edit), src: edit_button_work_path(work), target: "_top" %>
-    <%= helpers.turbo_frame_tag dom_id(work, :delete), src: delete_button_work_path(work) %>
+    <%= helpers.turbo_frame_tag dom_id(work, :edit_in_progress), src: edit_button_work_path(work, tag: "in_progress"), target: "_top" %>
```
Comment (mjgiarlo) [inline app/components/dashboard/in_progress_row_component.html.erb:4]: Non-unique IDs bad.

File: app/components/works/purl_reservation_modal_component.html.erb [hunk 4]
```diff
@@ -1,8 +1,8 @@
-<div class="modal fade" id="purlReservationModal" tabindex="-1" aria-labelledby="content-type-prompt" aria-hidden="true">
+<div class="modal fade" id="purlReservationModal" tabindex="-1" aria-labelledby="deposit-title-prompt" aria-hidden="true">
```
Comment (mjgiarlo) [inline app/components/works/purl_reservation_modal_component.html.erb:1]: Non-unique IDs (due to copypasta) bad.

File: app/views/collections/_dashboard.html.erb [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <%= turbo_frame_tag dom_id(collection, :summary) do %>
-  <li class="mb-5"><!-- from _dashboard.html -->
+  <div class="mb-5"><!-- from _dashboard.html -->
```
Comment (mjgiarlo) [inline app/views/collections/_dashboard.html.erb:2]: No need to have a `ul/li` structure for this, AFAICT.

File: app/views/dashboards/show.html.erb [hunk 6]
```diff
@@ -3,6 +3,7 @@
 <% end %>
 <main class="mb-3 mb-md-5" id="content">
   <div class="container px-4 px-md-3" id="dashboard" data-controller="work-type work-type-modal">
+    <h1 class="mb-3">Dashboard</h1>
```
Comment (mjgiarlo) [inline app/views/dashboards/show.html.erb:6]: Lack of top-level headings bad.

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -28,11 +28,11 @@
   </head>
 
   <body data-controller="help-how">
-    <div class="overflow-hidden">
+    <nav class="overflow-hidden" aria-label="Skip to main content">
```
Comment (mjgiarlo) [inline app/views/layouts/application.html.erb:31]: Lack of aria bad. Prefer semantic tags.

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -83,14 +83,14 @@
           </div>
         </nav>
         <% if controller_name != 'welcome' %>
-          <div class="background-holder">
+          <header class="background-holder">
```
Comment (mjgiarlo) [inline app/views/layouts/application.html.erb:86]: Prefer semantic tags.

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -120,38 +120,38 @@
           </div>
         </div>
       </div>
-    </footer>
 
-    <div id="global-footer" role="contentinfo">
```
Comment (mjgiarlo) [inline app/views/layouts/application.html.erb:125]: Multiple `role="contentinfo"` elements bad. Move the SU footer into the HTML footer.

File: app/views/shared/_terms_of_deposit.html.erb [hunk 10]
```diff
@@ -1,9 +1,9 @@
-<div class="modal" id="termsOfDepositModal" tabindex="-1" aria-labelledby="content-type-prompt" aria-hidden="true">
+<div class="modal" id="termsOfDepositModal" tabindex="-1" aria-labelledby="terms-of-deposit-prompt" aria-hidden="true">
```
Comment (mjgiarlo) [inline app/views/shared/_terms_of_deposit.html.erb:1]: Non-unique IDs (due to copypasta) bad.

File: app/views/welcome/first_time.html.erb [hunk 11]
```diff
@@ -1,9 +1,9 @@
-<div class="row justify-content-center">
+<main class="row justify-content-center" id="content">
```
Comment (mjgiarlo) [inline app/views/welcome/first_time.html.erb:1]: Prefer semantic elements. And provide an anchor for the skip link.

File: app/views/welcome/first_time.html.erb [hunk 12]
```diff
@@ -1,9 +1,9 @@
...
   <div class="card col-md-10 col-lg-8 col-xl-7 col-xxl-6 mt-5 mb-5">
     <div class="card-body">
-      <h5 class="card-title">Welcome!</h5>
+      <h1 class="card-title">Welcome!</h1>
```
Comment (mjgiarlo) [inline app/views/welcome/first_time.html.erb:4]: Select a heading based on semantics, not display (which can be controlled by CSS).

File: app/views/welcome/show.html.erb [hunk 13]
```diff
@@ -18,7 +18,8 @@
     <a href="#photo-credit" class="credit">Photo credit<sup>*</sup></a>
   </div>
 </main>
-<div class="benefits">
+
+<section class="benefits" aria-label="Benefits">
```
Comment (mjgiarlo) [inline app/views/welcome/show.html.erb:22]: Prefer semantic elements.

File: app/views/welcome/show.html.erb [hunk 14]
```diff
@@ -67,22 +68,22 @@
       </div>
     </div>
   </div>
-</div>
+</section>
 
-<div class="whys">
+<section class="whys" aria-label="Frequently Asked Questions">
```
Comment (mjgiarlo) [inline app/views/welcome/show.html.erb:73]: Prefer semantic elements.

File: app/views/welcome/show.html.erb [hunk 15]
```diff
@@ -67,22 +68,22 @@
...
   <div class="container p-4">
     <div class="row">
       <div class="col-md-3">
-        <%= image_tag "home_who.jpg", height: 80, width: 80, aria: {hidden: true} %>
+        <%= image_tag "home_who.jpg", height: 80, width: 80, aria: {hidden: true}, alt: "Who" %>
```
Comment (mjgiarlo) [inline app/views/welcome/show.html.erb:77]: Use `alt` attrs consistently.

**Review by ndushay — APPROVED** [review]
LGTM, modulo Amy approving visible changes.

**Pull request merged**